src/screens/Landing/Subscribe/style.scss

```diff
@@ -13,24 +13,27 @@
     left: 9rem;
     display: flex;
     flex-direction: column;
+    margin: 0;
 
     .title {
       font-size: 2.5rem;
       font-family: $font-primary;
+      text-align: center;
```

Comment:
This helps a lot only when in mobile view, so maybe place it in the media query...?